config/management/README.md

```diff
@@ -70,74 +70,89 @@ While this is compiled as a single binary it provides several different faciliti
+done is outside the scope of this document.
+
+The remainder of this section specifies distinct behaviors for each role.
+
+### The Association
+
+* The association will publish a layout containing the distinct names and roles
+  of the participants, this is placed into a common namespace:
+```
+management set-layout \
+  --path PATH_TO_LAYOUT \
+  --backend 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=common'
+```
+* Each Member of the Association will upload their key to GitHub:
 ```
-management association-key \\
-    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+management association-key \
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \
     --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
 ```
-  * Owner:
+
+### Validator Owners
+
+* Each Validator Owner member will upload their key to GitHub:
 ```
-management owner-key \\
-    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+management owner-key \
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \
     --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
 ```
-  * Operator:
+
+### Validator Operators
+
+* Each Validator Operator member will upload their key to GitHub:
 ```
-management association-key \\
-    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \\
+management operator-key \
+    --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \
     --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
 ```
-* The operator will upload a signed validator-config (note: the owner address
-  is irrelevant in this run):
+* For each, validator managed by an operator, the operator will upload a signed
+  validator-config. The namespace in GitHub correlates to the owner namspace
+  (note: the owner address is irrelevant in this run):
 ```
-management validator-config \\
+management validator-config \
     --owner-address 00000000000000000000000000000000 \
     --validator-address '/dns/DNS/tcp/PORT' \
     --fullnode-address '/dns/DNS/tcp/PORT' \
     --local 'backend=vault;server=URL;token=PATH_TO_TOKEN' \
     --remote 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=NAME'
 ```
-* The association will publish a layout containing the distinct names and roles
-  of the participants, this is placed into a common namespace:
-```
-management set-layout \\
-  --path PATH_TO_LAYOUT \\
-  --backend 'backend=github;owner=OWNER;repostiory=REPOSITORY;token=PATH_TO_TOKEN;namespace=common'
-```
-* Each participant can now produce genesis, this requires no namespace:
+* Upon receiving signal from the association, validator operators can now build
+  genesis, this requires no namespace:
 ```
-management genesis \\
-  --path PATH_TO_LAYOUT \\
+management genesis \
+  --path PATH_TO_LAYOUT \
```

Comment:
Where do they get PATH_TO_LAYOUT from?

Reply:
they generate a layout locally and ..

since we'll be doing this in the immediate future, I'll leave out the details :)